api/src/context.rs

```diff
@@ -133,10 +133,10 @@ impl Context {
     ) -> Result<Vec<TransactionOnChainData>> {
         let data = self
             .db
-            .get_transactions(start_version, limit as u64, ledger_version, true)?;
+            .get_transaction_outputs(start_version, limit as u64, ledger_version)?;
```

Comment:
Are there use cases that do care about only transactions and not the corresponding output? If so, does it make sense for us to provide a separate API for getting transactions with output and leaving the current lightweight API intact especially because the write sets for txns can be large.

Reply:
potentially we can move this to requiring a query param to return the full write set, but testing across 500 it's still minimal performance impact relative to not having it